apps/docs/docusaurus.config.js

```diff
@@ -31,6 +31,23 @@ const config = {
         blog: false
       }),
     ],
+    [
+      'redocusaurus',
+      {
+        // Plugin Options for loading OpenAPI files
+        specs: [
+          {
+            // spec: path.join(__dirname, "api-spec.json"),
+            spec:'http://127.0.0.1:4200/api/openapi',
```

Comment:
This should use the HOST variable instead of the local network.

Use a ternary and replace 127.0.0.1:4200 with it.

Check whether one of the SSL variables are defined, if it is, add an S onto the end of HTTP.